presto-spi/src/main/java/com/facebook/presto/spi/page/PagesSerdeUtil.java

```diff
@@ -60,6 +66,17 @@ public static void writeSerializedPage(SliceOutput output, SerializedPage page)
         output.writeByte(page.getPageCodecMarkers());
         output.writeInt(page.getUncompressedSizeInBytes());
         output.writeInt(page.getSizeInBytes());
+        if (CHECKSUMMED.isSet(page.getPageCodecMarkers())) {
+            output.writeLong(page.getChecksum());
+            try {
+                String host = Inet6Address.getLocalHost().getHostAddress();
+                output.writeInt(host.length());
+                output.writeBytes(host.getBytes(StandardCharsets.UTF_8));
+            }
+            catch (UnknownHostException e) {
+                e.printStackTrace();
```

Comment:
use a Logger here

Reply:
import com.facebook.airlift.log.Logger;
is not available in presto_spi.  Can I depend on that?